apps/web/src/pages/InstructorLedTrainingPage/InstructorLedTrainingPage.tsx

```diff
@@ -0,0 +1,545 @@
+            id: "yd+VEB",
+            description: "Label for training date meta data",
+          })}
+          {intl.formatMessage(commonMessages.dividingColon)}
+          <span data-h2-font-weight="base(bold)">
+            {formatDate({
+              date: parseDateTimeUtc(trainingOpportunity.trainingStart),
+              formatString: "PPP",
+              intl,
+            })}
+          </span>
+        </>
+      ),
+    } satisfies MetaDataItemProps);
+  }
+
+  if (trainingOpportunity.trainingStart && trainingOpportunity.trainingEnd) {
+    metaData.push({
+      key: `training-dates-${trainingOpportunity.id}`,
+      type: "text",
+      children: (
+        <>
+          {intl.formatMessage({
+            defaultMessage: "Training dates",
+            id: "yK0vlP",
+            description: "Label for training dates meta data",
+          })}
+          {intl.formatMessage(commonMessages.dividingColon)}
+          <span data-h2-font-weight="base(bold)">
+            {formatDate({
+              date: parseDateTimeUtc(trainingOpportunity.trainingStart),
+              formatString: "PPP",
+              intl,
+            })}{" "}
+            {"-"}{" "}
+            {formatDate({
+              date: parseDateTimeUtc(trainingOpportunity.trainingEnd),
+              formatString: "PPP",
+              intl,
+            })}
+          </span>
+        </>
+      ),
+    } satisfies MetaDataItemProps);
+  }
+
+  if (trainingOpportunity.registrationDeadline) {
+    metaData.push({
+      key: `course-format-${trainingOpportunity.id}`,
+      type: "text",
+      children: (
+        <>{getLocalizedName(trainingOpportunity.courseFormat?.label, intl)}</>
+      ),
+    } satisfies MetaDataItemProps);
+  }
+
+  return metaData;
+}
+
+const selectedFilterStyle: Record<string, string> = {
+  mode: "inline",
+  color: "secondary",
+  "data-h2-text-decoration": "base(none)",
+};
+
+const unselectedFilterStyle: Record<string, string> = {
+  mode: "inline",
+  color: "black",
+  "data-h2-font-weight": "base(bold)",
+};
+
+const filterTrainingOpportunities = (
+  trainingOpportunities: TrainingOpportunity[],
+  filterBy?: CourseLanguage.English | CourseLanguage.French | null,
+): TrainingOpportunity[] => {
+  if (!filterBy) {
+    return trainingOpportunities;
+  }
+  const filteredTrainingOpportunities = trainingOpportunities.filter(
+    (trainingOpportunity) => {
+      return trainingOpportunity.courseLanguage?.value === filterBy;
+    },
+  );
+  return filteredTrainingOpportunities;
+};
+
+export const Component = () => {
+  const intl = useIntl();
+  const paths = useRoutes();
+
+  const [trainingOpportunitiesState, setTrainingOpportunitiesState] = useState<
+    TrainingOpportunity[]
+  >([]);
+  const [trainingOpportunitiesFilteredBy, setTrainingOpportunitiesFilteredBy] =
+    useState<CourseLanguage.English | CourseLanguage.French | null>(null);
+
+  const [{ data, fetching }] = useQuery({
```

Comment:
Could we set this to page size 100 or 500 or something?  Since we don't have paging controls let's try to get as many as possible on initial load.

Reply:
Good point! I increased it to 100 in [f0155e1](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12083/commits/f0155e1b8cf852da22cfbf84d2a2935ad7ed8418)